ssl/ssl_lib.c

```diff
@@ -2081,20 +2089,26 @@ long SSL_CTX_ctrl(SSL_CTX *ctx, int cmd, long larg, void *parg)
     case SSL_CTRL_SET_SESS_CACHE_SIZE:
         if (larg < 0)
             return 0;
-        l = (long)ctx->session_cache_size;
-        ctx->session_cache_size = (size_t)larg;
+        session_cache = ctx->session_cache;
+        l = (long)session_cache->size;
+        session_cache->size = (size_t)larg;
         return l;
     case SSL_CTRL_GET_SESS_CACHE_SIZE:
-        return (long)(ctx->session_cache_size);
+        return (long)(ctx->session_cache->size);
     case SSL_CTRL_SET_SESS_CACHE_MODE:
-        l = ctx->session_cache_mode;
-        ctx->session_cache_mode = larg;
+        session_cache = ctx->session_cache;
+        l = session_cache->mode;
+        session_cache->mode = larg;
         return (l);
```

Comment:
If you have styling issue in near vicinity, fix it. I mean it should be `return l;`, right?